tools/check_docs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
I won't repeat comments that I've made in PR#41

Reply:
Still to do

Reply:
All resolved